src/features/landing/components/ForWhoSection/styles.ts

```
@@ -1,5 +1,34 @@
 import breakpointsTheme from '@/components/UiBreakpoints';
 import colorTheme from '@/components/UiColorTheme';
+import { media } from '@/components/UiBreakpoints/Theme/media';
```

ESLint

[error] `@/components/UiBreakpoints/Theme/media` import should occur before import of `@/components/UiColorTheme`

(import/order)